### PR TITLE
rest, spread and bindKeys

### PR DESCRIPTION
**Pull request by cedmax**
I oversimplified bindKeys to remove the placeholder example, but I think it's ok, considering it's the same semplification we used in the _.bind method

### CODE REVIEW

File: src/content/lodash/function/bindKey/lodash.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// https://lodash.com/docs/#bindKey
+import _, { bindKey } from 'lodash'
```
Comment (102) [inline src/content/lodash/function/bindKey/lodash.js]: Does it make sense to omit the `_` import (and/or mention directly in a comment in the code that placeholder option isn't implemented in the replacement?)

```suggestion
import { bindKey } from 'lodash'
```

**Review by 102 — DISMISSED** [review]
Both rest & spread fns have optional `start` argument, but I'm not sure it's worth implementing them by default -- at least I wasn't able to find a quick/elegant solution about their implementation (and I definitely haven't seen them being used in the real life 🙂)

**Comment by cedmax** [comment]
> I'm not sure it's worth implementing them

I'm going to stick with the "this isn't meant to be a drop in replacement" position on this one: I"d rather not overcomplicate the vanilla version to mimic the api 1:1, but focus on the underlying behaviour to "explain" it

File: src/content/lodash/function/bindKey/lodash.js (hunk 1 above)
Comment (cedmax) [reply src/content/lodash/function/bindKey/lodash.js]: Definitely, this is a leftover of when I was using other lodash methods in the method, will push the change and merge, thanks

Comment (cedmax) [reply src/content/lodash/function/bindKey/lodash.js]: I'm a bit surprised it wasn't caught by lint though: I moved to `next lint` as part of the transition to next.js but I might have to refine the rules

**Pull request merged**